joblib/test/test_hashing.py

```diff
@@ -403,6 +403,13 @@ def test_hashes_are_different_between_c_and_fortran_contiguous_arrays():
 
 
 @with_numpy
+def test_array_with_empty_shape():
```

Comment:
Just do a simple test for 0d array  similarly to test_hash_object_dtype (and call it something like test_0d_array).